site/docs/concepts/web-app.md

```diff
@@ -242,18 +222,103 @@ There are three ways to begin creating a materialization:
 When you initiate the workflow in any of these ways, all available materialization connectors are displayed.
 Select a connector to reveal the full form with configuration options specific to your desired destination.
 
-Fill out the **Endpoint Config** form and use the **Collection Selector** to map Flow collections to
-resources in the destination system.
+Fill out the **Endpoint Config** form and in the **Source Collections** section use the **Collection Selector** to map Flow collections to resources in the destination system.
+
+Note that if you entered the workflow from the Captures page or after publishing a capture, collections will be pre-populated for you.
```

Comment:
Adding in a mention of the "Source Collections" title to hope people connect the sections mentally.

Eventually I'd like to document this entire section